server/broadcast.go

```diff
@@ -462,6 +479,35 @@ func shouldStopSession(err error) bool {
 	return sessionErrRegex.MatchString(err.Error())
 }
 
+func verify(verifier *verification.SegmentVerifier,
+	cxn *rtmpConnection, sess *BroadcastSession,
+	source *stream.HLSSegment, res *net.TranscodeData) error {
+	params := &verification.VerifierParams{
+		ManifestID:   sess.ManifestID,
+		Source:       source,
+		Profiles:     sess.Profiles,
+		Orchestrator: sess.OrchestratorInfo,
+		Results:      res,
+	}
+
+	// Verify. The params we receive, if any, are the *accepted* params
+	// The accepted params are not necessarily the same params we just have!
+	p, err := verifier.Verify(params)
+	if _, retry := err.(verification.Retryable); retry {
```

Comment:
Is the plan to just check for `verification.Retryable` errors for now and consider distinguishing between fatal and verifier related errors later (as described in #1100)?

Reply:
Good point, I missed that the spec distinguishes between verifier errors and other types of fatal verification-related errors. I had been lumping them all under "retryable" errors (called such because a little weird to be retrying on supposed "fatal" errors).

The spec does imply that verifier errors shouldn't immediately remove the orchestrator from the working set, while fatal errors should - which is probably sensible and the reason for making a distinction between the two. Will adjust.